tests/unit/versioning/VersionID.js

```diff
@@ -0,0 +1,50 @@
+const VID = require('../../../lib/versioning/VersionID.js');
+const assert = require('assert');
+
+function randkey(length = 15) {
```

Comment:
This is the comment I like; it touches the right issues + having convincing reasoning for its suggestions.